demisto_sdk/commands/validate/initializer.py

```diff
@@ -254,33 +253,28 @@ def get_unfiltered_changed_files_from_git(self) -> Tuple[Set, Set, Set]:
             If this command runs on a build triggered by an external contribution PR,
             the relevant modified files would have an "untracked" status in git.
             The following code segment retrieves all relevant untracked files that were changed in the external contribution PR
-            and adds them to `modified_files`. See CIAC-10490 for more info.
+            See CIAC-10968 for more info.
             """
             logger.info(
-                "\n[cyan]CONTRIB_BRANCH variable found, trying to collected changed untracked files from external contribution PR[/cyan]"
+                "\n[cyan]CONTRIB_BRANCH environment variable found, running validate in contribution flow "
+                "on files staged by Utils/update_contribution_pack_in_base_branch.py (Infra repository)[/cyan]"
             )
+            # Open contribution_files_paths.txt created in Utils/update_contribution_pack_in_base_branch.py (Infra) and read file paths
+            relative_untracked_files_paths: Set[Path] = set()
+            with open(
+                "contribution_files_relative_paths.txt", "r"
+            ) as contribution_files:
+                for line in contribution_files:
+                    clean_line: str = line.rstrip("\n")
+                    relative_untracked_files_paths.add(Path(clean_line))
```

Comment:
In this code segment, we are iterating over lines in a file, so Its not a `single_file`, but a `single_line`.
I did mistakenly named `contribution_files` instead of `contribution_file`, which would made it much clearer.
Fixing this now.